src/style.css

```diff
@@ -1,3 +1,78 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --light-blue: rgb(0, 172, 220);
+  --white: rgb(255, 255, 255);
+}
+
+* {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
 body {
+  padding: 0;
+  max-width: 1200px;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+}
+
+header {
+  background: var(--white);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 10px 50px;
+}
+
+ul {
+  list-style-type: none;
   margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: rgb(0, 0, 0);
+}
+
+.nav__link:hover {
+  color: var(--light-blue);
+}
+
+.logo {
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__wrapper {
+  display: flex;
+}
+
+.nav__item {
+  position: relative;
+  margin-right: 20px;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.item:last-child {
+  margin-right: 0;
+}
+
+.is-active {
+  color: var(--light-blue);
+}
+
+.is-active:after {
+  content: "";
+  position: absolute;
+  border-radius: 8px;
+  left: 0;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)